.github/workflows/assignproj.yml

```diff
@@ -4,8 +4,10 @@ on:
   issues:
     types: [opened]
 
+permissions: write-all
+
 env:
-  MY_GITHUB_TOKEN: ${{ secrets.RUBY_GITHUB_TOKEN }}
```

Comment:
`RUBY_GITHUB_TOKEN` is only used here. `GITHUB_TOKEN` takes care of this so we don't need both.